livestream-compose-sample/src/main/java/io/getstream/livestream/compose/ui/LiveStreamChannels.kt

```diff
@@ -80,85 +65,62 @@ fun LiveStreamChannels(
+    // We load a [LazyVerticalGrid] when we have a grid view to show
+    if (isGrid) {
+        LazyVerticalGrid(
+            modifier = modifier,
+            cells = GridCells.Fixed(2),
+            contentPadding = PaddingValues(bottom = 12.dp, top = 12.dp),
+            content = {
+                items(channels) { item ->
+                    LiveStreamChannelCard(
                         modifier = Modifier
-                            .height(240.dp)
-                    ) {
-                        Row(
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .height(105.dp)
-                        ) {
-                            Image(
-                                painterResource(channels[index].channelArt),
-                                contentDescription = stringResource(id = R.string.accessibilityArtworkImage),
-                                contentScale = ContentScale.Crop,
-                            )
-                        }
-                        Row(
-                            modifier = Modifier
-                                .fillMaxWidth()
-                                .padding(start = 8.dp, end = 8.dp)
-                                .wrapContentHeight(),
-                            verticalAlignment = Alignment.Top
-                        ) {
-                            Icon(
-                                imageVector = Icons.Rounded.ViewStream,
-                                contentDescription = stringResource(id = R.string.accessibilityIcon)
-                            )
-                            Text(
-                                modifier = Modifier
-                                    .align(Alignment.CenterVertically)
-                                    .padding(start = 4.dp),
-                                color = ChatTheme.colors.textHighEmphasis,
-                                text = channels[index].channelName
-                            )
+                            .fillMaxWidth()
+                            .padding(10.dp),
+                        isDarkTheme = isDarkTheme,
+                        liveStreamChannelItem = item,
+                        onCardClick = { clickedItem ->
+                            openLiveStream(context, clickedItem)
                         }
-                        Text(
-                            modifier = Modifier
-                                .padding(8.dp),
-                            text = context.resources.getString(channels[index].channelDescription),
-                            maxLines = 3,
-                            color = ChatTheme.colors.textLowEmphasis,
-                            overflow = TextOverflow.Ellipsis
-                        )
-                    }
+                    )
                 }
             }
+        )
+    } else {
+        // We load a [LazyColumn] for when list is required
+        LazyColumn {
```

Comment:
We should be using the MessageList here instead of a `LazyColumn` - we discussed how the XML applications we have don't make sense because they don't use our components, but this is the same approach.

Reply:
Why `messageList` for channels ? 

Anyways updated `LazyColumn` to our ChannelList component